fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+
+
+      In addition to the HTTP status codes, responses will also use Location and Retry-After headers to specify the polling URI endpoint and polling frequency respectively.
+      
+      
+      The response will contain the partially calculated draft order and any calculated attributes dependant on stepwise operations will be null/empty until available.
+      
+      For long running operations, clients will be responsible for polling the endpoint specified in the Location header until a 200 OK response is returned.
+      
+    x-traitTag: true  
+    
+    
+
+paths:
+  /admin/draft_orders.json:
+    get:
+      summary: Retrieve a list of draft orders
+      description: Return a list of draft order objects.
+      parameters:
+      - $ref: '#/parameters/fields'
+      - $ref: '#/parameters/limit'
+      - $ref: '#/parameters/page'
+      - $ref: '#/parameters/since_id'
+      - $ref: '#/parameters/status'
+      - $ref: '#/parameters/ids'
+    
+      tags:
+        - Get
+
+      responses:
+        '200':
+          description: List of draft orders
+          schema:
+            $ref: '#/definitions/DraftOrder'
+        
+        '400':
+          description: Bad request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                 
+       
+          
+            
+      x-code-samples:
+        - lang: curl
+          source:  |
+            curl -X GET -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders.json
+            
+        
+            
+      
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              draft_order = ShopifyAPI::DraftOrder.all
+            
+            
+  
+    post:
+      summary: Create a draft order
+      description: |
+          
+          Using the Draft Orders API you can create orders in draft state using product variant line items, or your own custom line items.  This enables you to create one-off products that exist outside of your regular inventory management system.  To create a product variant draft order, simply specify a `variant_id` `quantity` and `discount` (if applicable).  Create a custom line item by supplying `title`, `price`, `taxable` and `quantity`. 
+          
+        
+          | `Note`| The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.  |
+          |---|-------------|
+          
+          #### About Custom Shipping Lines
```

Comment:
formatting issue: could use more spacing between this heading and the preceding table:

![screen shot 2016-11-28 at 1 43 22 pm](https://cloud.githubusercontent.com/assets/10617892/20681221/a73ae1ba-b570-11e6-8618-85c5bf7f5b6e.png)